src/Sylius/Bundle/AdminBundle/Resources/assets/controllers/CompoundFormErrorsController.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { Controller } from '@hotwired/stimulus';
+
+/* stimulusFetch: 'lazy' */
+export default class extends Controller {
+    tabErrorBadgeClasses = 'tab-error';
+    accordionErrorBadgeClasses = 'accordion-error';
+    observer;
+
+    initialize() {
+        super.initialize();
+
+        this.observer = new MutationObserver(() => {
+            this.reloadBadges();
+        });
+    }
+
+    connect() {
+        super.connect();
+
+        this.updateFormErrors();
+        this.observe();
+    }
+
+    disconnect() {
+        super.disconnect();
+
+        this.observer.disconnect();
+    }
+
+    observe() {
+        this.observer.observe(this.element, { attributes: false, childList: true, subtree: true });
+    }
+
+    reloadBadges() {
+        this.observer.disconnect();
+        this.clearBadges();
+        this.updateFormErrors();
+        this.observe();
+    }
+
+    clearBadges() {
+        this.element.querySelectorAll(".tab-error").forEach(el => el.remove());
+        this.element.querySelectorAll(".accordion-error").forEach(el => el.remove());
+    }
+
+    updateFormErrors() {
+        this.updateErrorBadges(this.element.querySelector('[role="tablist"]'), this.tabErrorBadgeClasses);
+        this.updateErrorBadges(this.element.querySelector('div.accordion'), this.accordionErrorBadgeClasses);
+    }
+
+    updateErrorBadges(controlElementsContainer, badgeClasses) {
+        if (null === controlElementsContainer) {
+            return;
+        }
+
+        const document = controlElementsContainer.ownerDocument;
+        controlElementsContainer.querySelectorAll('button[type="button"][data-bs-toggle]').forEach((controlElement) => {
+            const errorsCount = this.countErrors(controlElement);
+            if (errorsCount > 0) {
+                const errorElement = document.createElement('div');
+                errorElement.classList.add(...badgeClasses.split(' '));
```

Comment:
```suggestion
        const document = controlElementsContainer.ownerDocument;
        const badgeClassesList = badgeClasses.split(' ');
        controlElementsContainer.querySelectorAll('button[type="button"][data-bs-toggle]').forEach((controlElement) => {
            const errorsCount = this.countErrors(controlElement);
            if (errorsCount > 0) {
                const errorElement = document.createElement('div');
                errorElement.classList.add(...badgeClassesList);
```

Reply:
Refactored to use a singular class so splitting is no longer necessary, the list was a leftover from an earlier attempt.